taxcalc/functions.py

```diff
@@ -551,7 +554,11 @@ def ItemDed(e17500_capped, e18400_capped, e18500_capped,
                  ID_StateLocalTax_c[MARS - 1])
     c18500 = min((1. - ID_RealEstate_hc) * e18500_capped,
                  ID_RealEstate_c[MARS - 1])
-    c18300 = c18400 + c18500
+    if ID_StateLocalTax_crt < 1:
```

Comment:
I don't think you should end up needing a conditional here given that you set ID_StateLocalTax_crt at 1 under current law.

Reply:
Thanks for the comments, Matt! I included the conditional to take care of rare cases where one's state and local tax deductions exceed their AGI (I think this happens when someone can write off a large business loss). I had considered setting the parameter at 9e99 under current law, but this still wouldn't take care of cases of itemizers with negative AGI. I'm more than open to suggestions which take care of the if condition.